docs/step.md

```diff
@@ -46,6 +46,10 @@ Check [react-floater](https://github.com/gilbarbara/react-floater) for more info
 **placementBeacon** {string} ▶︎ placement  
 The placement of the beacon. It will use the placement if nothing is passed and it can be: `top, bottom, left, right`.
 
+**spotlightTarget** {Element\|string} ▶︎ target
+The target for the step spotlight. It can be a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors) or an HTMLElement directly \(but using refs created in the same render would required an additional render afterwards\).
```

Comment:
The description isn't very clear.
I think it should be more like: "Define an optional element inside your step's `target` to be the actual target for the tooltip. The beacon would still use the `target`"

Reply:
As I wrote above, `spotlightTarget` is not an actual target for the tooltip, but the only target for the spotlight.